libmcount/dynamic.c

```diff
@@ -127,15 +129,25 @@ static int do_dynamic_update(struct symtabs *symtabs, char *patch_funcs)
 				stats.skipped++;
 				break;
 			case 0:
+				success = true;
 			default:
 				break;
 			}
 			stats.total++;
 		}
 
+		if (!success) {
+			nr_failed++;
+			func_name = name;
+		}
+
 		name = strtok(NULL, ";");
 	}
 
+	if (nr_failed)
+		pr_out("%s cannot be patched dynamically\n",
+		       nr_failed > 1 ? "some functions" : func_name);
+
```

Comment:
But a weird result from the above PATCH can occur.
`dynamic:  patched:       -2 (0.00%)`

@namhyung So.. Is it better to add new member `stats.missing` ? 😟 
```
$ uftrace -v -P wrong_name1 -P wrong_name2 ./hello.dyn
uftrace: creating 1 thread(s) for recording
uftrace: using libmcount.so library for tracing
mcount: initializing mcount library
dynamic: dynamic update stats:
dynamic:    total:        0
dynamic:  patched:       -2 (0.00%)
dynamic:   failed:        0 (0.00%)
dynamic:  skipped:        2 (0.00%)
mcount: mcount setup done
mcount: shmem_finish: tid: 26486 seqnum = 0 curr = 0, nr_buf = 2 max_buf = 2
some functions cannot be patched dynamically
hello
foo
bar
uftrace: child terminated with exit code: 4
uftrace: reading /tmp/uftrace-live-Tmm1bj/task.txt file
uftrace: live-record finished.. 
uftrace: start live-replaying...
uftrace: reading /tmp/uftrace-live-Tmm1bj/task.txt file
# DURATION    TID     FUNCTION
   1.130 us [26486] | __monstartup();
   0.732 us [26486] | __cxa_atexit();
   1.383 us [26486] | puts();
   0.401 us [26486] | puts();
   0.170 us [26486] | puts();

```